test/unit/org/apache/cassandra/db/compaction/CompactionStrategyManagerPendingRepairTest.java

```diff
@@ -371,15 +365,6 @@ public void testFinalizedAndCompactionRace() throws IOException
         assertEquals(1, cfs.getLiveSSTables().size());
         assertEquals(1, cfs.getPendingRepairSSTables(repairID).size());
 
-        System.out.println("*********************************************************************************************");
```

Comment:
Nit: I personally find these printouts nice to have, as long as they wouldn't cause the test to fail for a different solution.

Reply:
I do also. But sometimes get told off for leaving prinouts in tests lol. Given we both agree let's leave it. Added back.